client/post-editor/editor-ground-control/style.scss

```diff
@@ -38,15 +38,38 @@
 	margin: 0 4px 8px;
 }
 
-.editor-ground-control__publish-button {
+.editor-ground-control .editor-publish-button {
+	@extend .is-primary;
+
 	border-radius: 4px 0 0 4px;
 	flex-grow: 1;
+
+	@include breakpoint( "<660px" ) {
+		display: none;
+	}
 }
+
 .editor-ground-control__time-button {
-	border-radius: 0 4px 4px 0;
+	@include breakpoint( ">660px" ) {
+		@include is-primary;
```

Comment:
This is awful. If you have a better alternative to making a button `primary` in desktop but normal in mobile, let me know.